docs/installation.rst

```diff
@@ -25,6 +25,9 @@ Installation on your system
   * ``cd pyradiomics``
   * ``sudo python -m pip install -r requirements.txt``
   * ``sudo python setup.py install``
+  
+  To use your build for interactive use and development:
+  * ``sudo python setup.py develop``
```

Comment:
Looks as it is consistent with previous step.


@JoostJM : I would then suggest to remove `sudo`, this is not a practice to encourage. Regular development should happen as regular user within a virtual environment.